### DISCUSSION TITLE
3.2.10升级3.5.7后，拉取依赖失败

### DISCUSSION THREAD

**Discussion by ShiChiYa7493** [NervJS/taro#12853] [Q & A] [answered]
升级之前一切正常，项目执行taro update project命令后，taro升级到了3.5.7。然后yarn install 或者npm install都统统失败。
网络无问题。
报错信息：
[1/4] Resolving packages...

info There appears to be trouble with your network connection. Retrying...

info There appears to be trouble with your network connection. Retrying...

info There appears to be trouble with your network connection. Retrying...

info There appears to be trouble with your network connection. Retrying...

info There appears to be trouble with your network connection. Retrying...

error Couldn't find package "hls.js@^1.1.5" required by "@tarojs/components@3.5.7" on the "npm" registry.

info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

**Comment by TheKonka** [comment, marked as answer]
你这貌似是网络问题
先用 `taro update self` 或者 `npm/pnpm/yarn i -g @tarojs/cli@3.5.7`  把 `taro` 升级到 `3.5.7`
https://github.com/NervJS/taro/pull/12126
然后可以参考这个升级指南做下调整
网络问题多试几次`install`
再有问题删除 `node_modules` 和 `lock`文件再 `install`
多尝试，可以解决的
